sources/advanced/Command.ts

```diff
@@ -14,10 +14,28 @@ export type Usage = {
     examples?: [string, string][];
 };
 
+/**
+ * The schema used to validate the Command instance.
+ *
+ * The easiest way to validate it is by using the [Yup](https://github.com/jquense/yup) library.
+ *
+ * @example
+ * yup.object().shape({
+ *   a: yup.number().integer(),
+ *   b: yup.number().integer(),
+ * })
+ */
+export type Schema<C extends Command<any> = Command<any>> = {
```

Comment:
Given that the intent is to make `Schema` a tool to improve typing, maybe we should drop the default value to ensure that users explicitly use CRTP here (instead of silently fallbacking to `any`), wdyt?

Reply:
Good idea!

Reply:
Done. I made `Command.Schema` default to `Command<BaseContext>`, since removing the default wouldn't make any difference there.